components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -80,8 +81,16 @@ internal object GlobalAccountManager {
         instance = null
     }
 
-    internal suspend fun authError(cause: Exception? = null) {
-        instance?.get()?.encounteredAuthError(cause)
+    internal suspend fun authError(operation: String) {
```

Comment:
Is there any chance of two concurrent threads of activity both discovering an auth error at the same time, resulting in two closely-spaced calls to this method without the opportunity to try a recovery?

Reply:
To close the loop - yes, and we've worked around this by allowing a few auth errors (but not too many) through.